python/ray/air/execution/resources/fixed.py

```diff
@@ -0,0 +1,148 @@
+from typing import Dict, Optional, List, Union
+
+from dataclasses import dataclass
+
+import ray
+from ray import SCRIPT_MODE, LOCAL_MODE
+from ray.air.execution.resources.request import (
+    ResourceRequest,
+    AllocatedResource,
+)
+from ray.air.execution.resources.resource_manager import ResourceManager
+
+
+# Avoid numerical errors by multiplying and subtracting with this number.
+# Compare: 0.99 - 0.33 vs (0.99 * 1000 - 0.33 * 1000) / 1000
+from ray.util.annotations import DeveloperAPI
+
+_DIGITS = 100000
+
+
+def _sum_bundle_resources(bundles: List[Dict[str, float]]) -> Dict[str, float]:
+    all_resources = {}
+    for resources in bundles:
+        for k, v in resources.items():
+            all_resources[k] = all_resources.get(k, 0) + v
+
+    return all_resources
+
+
+@DeveloperAPI
+@dataclass
+class FixedAllocatedResource(AllocatedResource):
+    bundles: List[Dict[str, float]]
+
+    def annotate_remote_objects(
```

Comment:
What is the intention of this method? Comment?

Reply:
can I call this method multiple times to annotate multiple objects?

Reply:
See annotation of parent class